recipes/tqdm/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "4.7.2" %}
+
+package:
+  name: tqdm
+  version: {{ version }}
+
+source:
+  fn: tqdm-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/t/tqdm/tqdm-{{ version }}.tar.gz
+  md5: 63dcfa4910aa10ed7bf5c6e4a961ad7f
+
+build:
+  entry_points:
+    - tqdm=tqdm._main:main
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    # - setuptools
```

Comment:
The setup.py of tqdm also has 

```python
try:
    from setuptools import setup
except ImportError:
    from distutils.core import setup
```
Which would allow setup.py to proceed without setuptools